fiftyone/core/map/process.py

```diff
@@ -246,17 +148,31 @@ def _do_update_samples(pool, pb, batches, batch_count, sample_count):
         pool.join()
 
 
-def _do_map_samples(pool, pb, batches, batch_count, queue):
+def _do_map_samples(
+    pool: multiprocessing.Pool,  # type: ignore
+    pb: fou.ProgressBar,
+    batches: List[fomb.SampleBatch],
+    batch_count: multiprocessing.Value,  # type: ignore
+    queue: multiprocessing.Queue,
+):
     num_batches = len(batches)
 
     with pool, pb:
-        pool.map_async(_map_batch, batches)
+        pool.map_async(
+            _map_batch,
+            (
+                (idx + 1, num_batches, batch)
+                for idx, batch in enumerate(batches)
+            ),
+        )
 
+        count = 0
         while True:
             try:
                 result = queue.get(timeout=0.01)
                 pb.update()
                 yield result
+                count += 1
```

Comment:
we don't use count anywhere?

Reply:
That might be a hold over from some testing I did. I'll make sure it's removed.